osis-core/src/main/java/com/vmware/osis/security/jwt/endpoint/RefreshTokenEndpoint.java

```diff
@@ -6,7 +6,7 @@
 package com.vmware.osis.security.jwt.endpoint;
 
 import com.vmware.osis.scality.AppEnv;
-import com.vmware.osis.scality.security.PlatformUserDetailsService;
+import com.vmware.osis.scality.security.ScalityUserDetailsService;
```

Comment:
Same tied with Nicolas's comment: https://github.com/scality/vmware-ose-scality/pull/9/files#r590070342

Otherwise, I am good with it.

Reply:
Yeah, I don't know where this is heading, but in any case, this repo should not include a single package in the `com.vmware` domain. We don't own it, as simple as that.

Reply:
Updated packages in the previous PR. Please re-review

Reply:
@NicolasT 
Please Approve this PR too and this also reflects the copyrights from earlier branch